features/counter.feature

```diff
@@ -1,25 +1,26 @@
-Uses: StepRegistry
+Uses: Web Sockets
 Uses: Selenium
 
 Feature: I can use the counter
 
-  #! StepRegistry start
   Feature-Start:
-    Given I open Chrome
+    Given I start a web socket server
+    And I open Chrome
+    And I navigate to http://localhost:9999
+    And I wait for the web socket client SimpleStepPublisher
 
   Background:
-    Given I navigate to http://localhost:9999
-    And StepRegistry client SimpleStepPublisher is connected
+    Given I reset the counter
 
   Scenario: I can read the counter
     Then the counter value is 0
 
   Scenario: I can increment the counter
-    Given the counter value is 0
+    And the counter value is 0
```

Comment:
Curious why you changed these to `And` rather than `Given`? I thought it read better with `Given`, thoughts?

Reply:
I think it reads better when the 'Background' steps are included. 
In ends up reading:

Given I reset the counter
And the counter value is 0

rather than 

Given I reset the counter
Given the counter value is 0

Reply:
👍